lib/tasks/remove_duplicate_virtual_users.rake

```diff
@@ -0,0 +1,49 @@
+namespace :remove_duplicate_virtual_users do
+  desc 'Remove duplicate users by redirecting all posts to a single user'
+  task :remove => :environment do
+    puts 'Removing duplicate users'
+
+    Post.all.find_each do |post|
+      next if post.receivers.empty?
+
+      post.receivers.each do |receiver|
+        next unless receiver.virtual_user
+
+        similar_users = User.where('name LIKE?', receiver.name)
```

Comment:
Is this actually matching other users? I'm not 100% sure what SQL this translates into, but if it's `SELECT * FROM users u if u.name LIKE 'Max Meijer'` then this is unlikely to match any other users (because there aren't any wildcards in the query).

Reply:
Please see the explanation below. 

There are in fact multiple virtual users with the exact same name because of the bug I described.

Reply:
Ah ok, the `LIKE` confused me a bit, as you might as well use `=` then. (Don't have to change this as far as I'm concerned, it can't hurt either).